install_config/install/prerequisites.adoc

```diff
@@ -363,6 +363,23 @@ $ dig node1.example.com @10.64.33.1 +short
 10.64.33.101
 ----
 
+*The Impact of Disabling dnsmasq*
+
+If you want to set up {product-title} with *dnsmasq* disabled (for example, if
+your *_/etc/resolv.conf_*  is managed by a configuration tool other than
+NetworkManager), then `openshift_use_dnsmasq=False` is set in the Ansible
+playbook.
```

Comment:
@ahardin-rh I think I'm a little confused by the above. Should it say:

"If you want to set up X, then set `openshift_use_dnsmasq` in the Ansible playbook to `false`."

or

"If you set up X, then {product-title} sets `openshift_use_dnsmasq` in the Ansible playbook to `false`"

?